src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form class="form"
+      action="#"
+      method="get"
+      data-qa="big"
+    >
+      <img
+        class="searchImage"
+        src="../src/images/Search.svg"
+        alt="Search"
+      >
```

Comment:
Don't put this image in html, use css background for it

here's an example
https://codepen.io/adskiy/pen/mdmLqXB